opencore.config.plist

```diff
@@ -58,7 +58,7 @@
 				<key>MaxKernel</key>
 				<string>21.99.99</string>
 				<key>MinKernel</key>
-				<string>17.0.0</string>
+				<string>20.4.0</string>
```

Comment:
Yep, this is correct as far as I can tell:
[macOS 11.3](https://opensource.apple.com/release/macos-113.html) > xnu-7195.101.1 > config > MasterVersion == 20.4.0